pkg/provider/cloud/kubevirt/network_policy.go

```diff
@@ -103,3 +117,19 @@ func reconcileClusterImagesNetworkPolicy(ctx context.Context, cluster *kubermati
 	}
 	return nil
 }
+
+// infrKubeVirtClusterPodCidr returns the infra KubeVirt cluster pod CIDR.
+func infrKubeVirtClusterPodCidr(ctx context.Context, client ctrlruntimeclient.Client) (string, error) {
```

Comment:
could you rename to `GetInfraPodCIDR`? Would be nice if in the naming we can stick to verbs like Get, Delete, Create etc.

Reply:
Done